packages/legacy/src/Plans/components/Wizard/MappingForm.tsx

```diff
@@ -159,21 +158,14 @@ export const MappingForm: React.FunctionComponent<IMappingFormProps> = ({
     };
   }) as OptionWithValue<Mapping>[];
 
-  const filteredAvailableTargets =
```

Comment:
1. in the UI there should be warning that the mapping is from different namespace (this is not a happy path)
2. alternatively we could show the mappings from different namespaces after toggling some switch/checkbox

Reply:
the namespace show in the list,
this is as expected for networkattachmentdefinition [1] , they are a "cluster resource" with namespace,
it's expected to refer to networkattachmentdefinition from different namespace.

It's also ok to use mappings and providers from different namespaces, it's just makes things very complicated

https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/docs/how-to-use.md#launch-pod-with-text-annotation-for-networkattachmentdefinition-in-different-namespace

Reply:
This is the bug fix, not surfacing the namespace here